data-serving/scripts/convert-data/convert_data.py

```diff
@@ -1,7 +1,7 @@
-'''
```

Comment:
AFAIK this script is only run manually so this wasn't needed, but I suppose consistency will act as a signal to us to remember to do logging this way in future.